mlrun/datastore/targets.py

```diff
@@ -1432,18 +1432,6 @@ def as_df(
 
 
 class MongoDBTarget(BaseStoreTarget):
```

Comment:
maybe add an example like you did for source? or perhaps all the targets are examples-less?